joomla/joomla-cms
# [4.0] com_cpanel. Add menu " Modules"

### Summary of Changes
- Adds menu "Modules" to "System" > "Manage" in BE.

### Testing Instructions
- See also images below.
- Joomla4 nightly build.
- Goto menu "System".
- "Modules" not present in category "Manage"
- Apply patch.
- "Modules" present in category "Manage"

## Before
![11-09-_2018_19-39-36](https://user-images.githubusercontent.com/20780646/45377339-97523a00-b5fa-11e8-878e-3c2ab3129e57.jpg)

## After
![11-09-_2018_19-38-06](https://user-images.githubusercontent.com/20780646/45377351-9faa7500-b5fa-11e8-89d4-b2e50ab291a9.jpg)
